ui/apps/routing/src/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { BehaviorSubject } from 'rxjs';
+import { MenuItemAreaType } from '@akashaproject/ui-awf-typings/lib/app-loader';
+
+interface RouteRepository {
+  all?: {
+    [key: string]: Record<string, string | string[]>;
+  };
+  active?: {
+    [key: string]: Record<string, unknown>;
+  };
+  byArea?: { [key in MenuItemAreaType]?: Array<Record<string, unknown>> };
+}
+
+export class RoutingPlugin {
+  static readonly routeRepository: RouteRepository = {
+    all: {},
+    active: {},
```

Comment:
so this property, `active` has 2 keys, `apps` and `widgets` which are lists of active appNames? My first impression was that it contains active menuItems... Can you make it more explicit?